Wiki.md

```diff
@@ -0,0 +1,251 @@
+        * `apt-get install -y iptables-persistent`
+        * `iptables -A INPUT -p tcp --dport 32400 -i [your-network-interface] -j DROP`
+
+-----
+
+#### Configure UnicornFFMPEG
+UnicornFFMPEG will replace the Plex built-in transcoder. To build UnicornFFMPEG, complete the following steps:
+1. `cd /opt`
+2. `git clone https://github.com/UnicornTranscoder/UnicornFFMPEG.git`
+3. `cd UnicornFFMPEG`
+4. `npm install`
+5. `LB_URL=UnicornLoadBalancer.example.com npm start`
+    * This will override the default localhost:port option with the FQDN of the loadbalancer. This environment variable is required for a working setup. 
+    * After this npm command finishes, a new Plex Transcoder binary will be saved to the ./bin folder of the repository. Copy the linux binary to your plex docker folder (or anywhere you want). In this example we're copying it to `/root/docker/plexv2/binaries/Plex Transcoder-linux`.
+
+In order to replace the built-in transcoder with this new one, we need to mount it into our plex container. Simply add the following mount to your plex container:
+```
+- "/root/docker/plexv2/binaries/Plex Transcoder-linux:/usr/lib/plexmediaserver/Plex Transcoder"
+```
+where `/usr/lib/plexmediaserver` is the binary location. Most mainstream plex containers use this path, such as the container released by LinuxServier.io. 
+
+With that new volume/file mount, restart plex to replace the built-in transcoder with the "fake" transcoder that interprets the transcode arguments.
+
+-----
+
+#### Configure UnicornTranscoder
+The UnicornTranscoder is where the actual magic of transcoding and serving a stream happens. This is the portion of software that you'll replicate on multiple nodes to distribute the transcoding jobs. Each node, or unicorntranscoder, will "check in" with the loadbalancer. The loadbalancer will then distribute the transcode, sessions, and downloads accordingly. 
+
+To install UnicornTranscoder, we will again configure it with envrionment variables. Since our example uses systemd service, we won't set them as global variables. The default variables are set in the `config.js` file. We'll override a few of them with our systemd Environment variables. 
+
+So, to install and configure UnicornTranscoder, complete the following steps:
+1. `cd /opt`
+2. `git clone https://github.com/UnicornTranscoder/UnicornTranscoder.git`
+3. `cd UnicornTranscoder`
+4. `npm install`
+5. `npm run install`
+
+The transcoder is now ready to run. Just like the loadbalancer, we're going to run this as a service. Be sure to substitute the envrionment variables for your own setup/FQDN:
+1. `nano /etc/systemd/system/unicorntranscoder.service`
```

Comment:
Maybe names of services should be consistent earlier UnicornLoadBalancer was camelcase

Reply:
This is the result of a "Find and Replace" to fix a spelling issue. Easy enough fix